cli/up.go

```diff
@@ -0,0 +1,26 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	upCmd = &cobra.Command{
+		Use:   "up <group> [scale]",
+		Short: "Bring a group up",
+		Long:  "Submit a group, with an optional scale, and start it",
+		Run:   upRun,
+	}
+)
+
+func upRun(cmd *cobra.Command, args []string) {
+	submitRun(cmd, args)
```

Comment:
so, if there were already 2 submitted units, you'd expect 2 units to be started, and then all 4 started?

i don't feel like this is consistent with the current behaviour of submit - submit 3 will _add_ 3, while it sounds like up 3 will _make sure there are 3_